src/templates/outer-layout.html

```diff
@@ -4,31 +4,31 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-    <link rel="apple-touch-icon" sizes="57x57" href="images/favicon/apple-icon-57x57.png">
```

Comment:
all the paths are relative so that test versions of the website can be run from a sub folder - eg http://qooxdoo.org/website/PR_15/